audio_pipeline.command

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Bash script for running the audio pipeline for a date folder
+
+# To test the files, do `path/to/audio_pipeline.command 2021-07-08`
+
+# The reddit scraper will download files to current date
+# The other scripts will use 2021-07-08 to test source and audio extraction
+# robustness
+
+# Use `chmod u+x audio_pipeline.command` if permission denied
+
+# Change to the directory containing the script
+cd "${0%/*}"
+
+PROGNAME=$0
+
+# Error message
+usage() {
+  cat << EOF >&2
+Usage: $PROGNAME [<date>]
+
+<date>: ...
+
+EOF
+  exit 1
+}
+# Check that there are at least 2 arguments
+if [ $# -lt 1 ]; then
+    echo "Error, you need to supply a date!"
+    usage
+    exit 1
+fi
+
+echo "Running audio pipeline"
+
+# Replace this with either the Conda or Python version
+"""
```

Comment:
I didn;t change this and mine ran.. which is weird right?

Reply:
it is also a little unclean, this is all a comment, where is the replacing needing to be done?

Reply:
> I didn;t change this and mine ran.. which is weird right?

I agree that it's weird; would this be because the environment is already activated from the requirements installation earlier?

Reply:
I've [updated the instructions] so that there is a line to replace.